tequilapi/client/dto.go

```diff
@@ -1,17 +1,24 @@
 package client
 
-// StatusDto holds connection status and session id
-type StatusDto struct {
+// StatusDTO holds connection status and session id
+type StatusDTO struct {
 	Status    string `json:"status"`
-	SessionId string `json:"sessionId"`
+	SessionID string `json:"sessionId"`
 }
 
-// IdentityDto holds identity address
-type IdentityDto struct {
+// StatisticsDTO holds statistics about connection
+type StatisticsDTO struct {
+	BytesSent       int `json:"bytesSent"`
```

Comment:
For external users, I would name it `uploadBytes` and `downloadBytes`

Reply:
`"uploadAmount": 5` is a bit confusing - it doesn't specify units. But I like `upload` - what do you think about `uploadedBytes` and `downloadedBytes` ?

Reply:
Applied naming we agreed on - renamed `durationSeconds` to `duration`, left byte fields unotuched.